sale_stock_available_info_popup/models/sale_order.py

```diff
@@ -16,15 +16,13 @@ class SaleOrderLine(models.Model):
     @api.depends("product_id", "product_uom_qty")
     def _compute_immediately_usable_qty_today(self):
         qty_processed_per_product = defaultdict(lambda: 0)
-        remaining = self.env["sale.order.line"]
+        self.immediately_usable_qty_today = False
         for line in self.sorted(key=lambda r: r.sequence):
             if not line.display_qty_widget:
-                remaining |= line
                 continue
             product = line.product_id
             qty_processed = qty_processed_per_product[product.id]
             line.immediately_usable_qty_today = (
                 product.immediately_usable_qty - qty_processed
             )
             qty_processed_per_product[product.id] += line.product_uom_qty
-        remaining.write({"immediately_usable_qty_today": False})
```

Comment:
@ernestotejeda a write should never be done in a `compute`, but `update`. I think this is also happening in v12.